### PR TITLE
[EventD] Cleanup RPC Servicer and separate validation logic for better clarity

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

## Summary
Separate out the validation logic into its own class for better readability.
No logical changes here.

## Test Plan
Run S1AP tests, EventD produces events

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by 119Vik** [comment]
RestartCWAGTests

File: orc8r/gateway/python/magma/eventd/event_validator.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+from typing import Any, Dict
+import pkg_resources
+import yaml
+
+from bravado_core.spec import Spec
+from bravado_core.validate import validate_object as bravado_validate
+
+class EventValidator(object):
+    """
+    gRPC based server for EventD.
```
Comment (hcgatewood) [inline orc8r/gateway/python/magma/eventd/event_validator.py:26]: Update docstring

**Review by hcgatewood — APPROVED** [review]
Lgtm, just fix the one docstring

File: orc8r/gateway/python/magma/eventd/event_validator.py (hunk 1 above)
Comment (themarwhal) [reply orc8r/gateway/python/magma/eventd/event_validator.py:26]: whoops auto merge :D will fix in following PRs

**Pull request merged**